server/sockets/cohort.js

```diff
@@ -170,6 +183,8 @@ const updateCohortHeaderStatus = (socket, cohortClientLocks) => {
     }, LOCK_TIMEOUT);
 
     cohortClientLocks.set(userId, delayedUnlock);
+
+    handleLocks(Cohort, { _id: cohortId })({ description, name });
```

Comment:
I would apply renames here as this leads to clearer intent: 
```javascript
const locks = { description: descriptionLock, name: nameLock };
handleLocks(Cohort, { _id: cohortId })(locks);
```

Consider the same pattern for line 151.